src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -41,17 +41,18 @@
   private final boolean enableRecursiveMacroCalls;
 
   private Map<Context.Library, Set<String>> disabled;
+  private final boolean strictUndefined;
```

Comment:
I think this could be named more descriptively like 'throwExceptionsForUnknownTokens' or 'failOnUnknownTokens'